app/views/posts/show.html.erb

```diff
@@ -26,5 +31,15 @@
         <h3>Like this post</h3>
         <%= render 'likes/form', like: @like %>
       </div>
```

Comment:
[OPTIONAL] The like button throws an error. you might want to look at that to make sure that everything in your app is functioning as expected.

![image](https://github.com/AyoMoses1/Gems_Blog/assets/38088818/70832d59-722d-4f01-97da-cff13925db99)

Reply:
This change has been done well